app/web/src/components/FuncEditor/FuncArguments.vue

```diff
@@ -12,54 +12,60 @@
         placeholder="New argument name..."
         :disabled="disabled"
       />
-      <SelectMenu
+      <!-- <SelectMenu
         v-model="newArg.kind"
         class="flex-auto"
         :options="kindOptions"
         :disabled="disabled"
-      />
+      /> -->
       <VButton
         label="Add"
         tone="success"
         :disabled="disabled"
-        @click="addArgument"
+        @click="createArgument"
       />
     </Inline>
     <ul>
-      <Inline v-for="arg in editingArgs" :key="arg.id" alignY="center">
+      <Inline v-for="arg in funcArguments" :key="arg.id" alignY="center">
         <VormInput
           :id="`arg-name-${arg.id}`"
           v-model="arg.name"
           type="text"
           noLabel
           placeholder="Argument name"
           :disabled="disabled"
-          @blur="saveArguments"
+          @blur="updateArgument(arg.id, arg.name, arg.kind, arg.elementKind)"
         />
-        <SelectMenu
+        <!-- <SelectMenu
```

Comment:
Fix these selects. Hopefully, we don't have to use `Option` and can mimic other select boxes.